graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -18,12 +18,12 @@ class BootstrapModalForm extends Component {
     this.open = this.open.bind(this);
     this.close = this.close.bind(this);
     this._submit = this._submit.bind(this);
-    this._onModalClose = this._onModalClose.bind(this);
+    this._onModalCancel = this._onModalCancel.bind(this);
   }
 
-  _onModalClose() {
-    if (typeof this.props.onModalClose === 'function') {
-      this.props.onModalClose();
+  _onModalCancel() {
+    if (typeof this.props.onCancel === 'function') {
```

Comment:
`onModalClose` is [still called](https://github.com/Graylog2/graylog2-server/blob/9d237670ae0d90ecae1ab35b0f9180fc7b1f9a8d/graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx#L70). We need to differentiate those two cases, because resetting the form is not required after submitting it, while it is necessary when cancelling it.